tasks/tests/test_prime_workflow.py

```diff
@@ -1,231 +1,233 @@
 # -*- coding: utf-8 -*-
-from locust.test.testcases import LocustTestCase
+# from locust.test.testcases import LocustTestCase
 
-from tasks import PrimeWorkflowTasks
-from locustfiles.prime_workflow import PrimeWorkflowUser
-from utils.constants import (
-    MOVE_TASK_ORDER,
-    MTO_SHIPMENT,
-    MTO_SERVICE_ITEM,
-    PAYMENT_REQUEST,
-)
-from utils.base import ImplementationError
+# from tasks import PrimeWorkflowTasks
```

Comment:
I think the old code in this file could also be removed instead of commented out